opal/mca/pmix/pmix4x/pmix/src/mca/bfrops/base/bfrop_base_fns.c

```diff
@@ -563,27 +563,28 @@ pmix_value_cmp_t pmix_bfrops_base_value_cmp(pmix_value_t *p,
             }
             rc = PMIX_EQUAL;
             break;
-        case PMIX_COORD:
-            rc = memcmp(p->data.coord, p1->data.coord, sizeof(pmix_coord_t));
-            if (0 > rc) {
+        case PMIX_COORD: {
```

Comment:
Ick - that sure violates a few of our coding standards. I don't really care about it here, though, as I can resolve that warning upstream and will overwrite any change made in OMPI anyway.

Reply:
Is it the use of the `{`? I can change it if you have a preferred way.

This one might be a bug (how it could be exposed I am not entirely sure). So it would be good to get this fixed right sooner than later.

Reply:
No worries - I'd leave it as-is. It will get overwritten next time I update PMIx which happens pretty frequently.

Reply:
What are we trying to fix here ?

Reply:
rc was defined as a `pmix_value_cmp_t` which can't store values < 0.

Reply:
indeed. why not use ret instead ?

Reply:
Good point - didn't notice ret, thanks. Fixed.